### ISSUE TITLE
Node missing from meta during plugin execution

### ISSUE THREAD

**Issue by brollb** [webgme/webgme-engine#268]
A node seems to be missing from the metamodel of one of the libraries during plugin execution.

The `Dropout` node seems to not be found during this code:
```
const meta = await core.getAllMetaNodes(rootNode);
return Object.values(meta)
    .find(child => core.getAttribute(child, 'name') === this.value);
```
This can be confirmed using the debugger:
![DeepinScreenshot_select-area_20220131151732](https://user-images.githubusercontent.com/4982789/151874374-0bf4634f-0e6a-4b9d-91f0-2b282f8e107d.png)

However, the node shows up in the metamodel:
![DeepinScreenshot_select-area_20220121093837](https://user-images.githubusercontent.com/4982789/151874005-f63e6d79-8488-4040-bd80-2f47a2b684eb.png)

@hneema08

**Comment by brollb** [comment]
I also confirmed that `Dropout` is a member of the `MetaAspectSet` for the library root (keras):
![DeepinScreenshot_select-area_20220131154240](https://user-images.githubusercontent.com/4982789/151877852-5c6fabea-035f-4e20-9a3d-663884d8ed05.png)

@kecso - Any other thoughts about why this might be happening?

**Comment by brollb** [comment]
Here is the library in question: https://github.com/deepforge-dev/deepforge-keras/raw/master/src/seeds/keras/keras.webgmex
